app/controllers/calendar_controller.rb

```diff
@@ -21,6 +21,7 @@ def index
   def create_calendar_props
     @props = {conferenceRooms: ConferenceRoom.all,
               initialEvents: events,
+              userEmail: session[:email],
```

Comment:
Things like this can really be a pain in the neck. But I think that `userEmail` is an example of a thing that if it is necessary it should be placed in React context. IMHO Passing it through a tree in this example is worse than setting up context.
Good call! 👍